src/block_cache/block_io.c

```diff
@@ -399,6 +420,16 @@ __wt_blkcache_write(WT_SESSION_IMPL *session, WT_ITEM *buf, uint8_t *addr, size_
     WT_STAT_SESSION_INCRV(session, bytes_write, dsk->mem_size);
     (void)__wt_atomic_add64(&S2C(session)->cache->bytes_written, dsk->mem_size);
 
+    if (dsk != NULL) {
+        if (dsk->type == WT_PAGE_COL_INT || dsk->type == WT_PAGE_ROW_INT) {
+            WT_STAT_CONN_INCRV(session, block_byte_write_intl, buf->size);
+            WT_STAT_CONN_INCRV(session, block_byte_write_intl_disk, dsk->mem_size);
```

Comment:
I think this is wrong. dsk->mem_size is the size before compression and encryption.

Reply:
I think you might be correct. I assumed they were the same because I saw a statement assigning dsk to ip->mem on line 415. However we didn't update dsk->size.